install-dev/init.php

```diff
@@ -128,6 +128,11 @@
             usort($themes, function ($a, $b) {
                 return strcmp($a, $b);
             });
+
+            if (empty($themes)) {
+                die('Error : Cannot find a valid theme in themes/ folder');
```

Comment:
```suggestion
                die('Error: Cannot find a valid theme in themes/ folder');
```

Reply:
Ok, nvm, I see it's a standard in this file 😜

Reply:
Well, the standard is a mistake... I totally agree with you, no space in English before colons. 👀

Reply:
Nice spotted @kpodemski and @Julievrz I fixed it